packages/postgrest-core/src/lib/get-table-from-url.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Parses a url and returns the table name the url is interacting with.
+ *
+ * For mutations, the .split('?') goes unused.
+ *
+ * @param url The url we are pulling the table name from
+ * @returns Table name
+ */
+export const getTableFromUrl = (url: string): string => {
+  const split = url.toString().split('/');
+  const table = split.pop() as string;
+  const maybeRpc = split.pop() as string;
+  return [maybeRpc === 'rpc' ? maybeRpc : null, table]
+    .filter(Boolean)
+    .join('/');
```

Comment:
Sorry, I am just on my phone but we still need to hand the query string like in the initial version  😅

Reply:
apologies! you are correct. thanks for the help!